src/services/auth/auth-router.ts

```diff
@@ -312,6 +312,37 @@ authRouter.get("/roles/", strongJwtVerification, async (_: Request, res: Respons
 	});
 });
 
+
+/**
+ * @api {get} /auth/roles/list/:role GET /auth/roles/list/:role
+ * @apiGroup Auth
+ * @apiDescription Get all users that have a certain role.
+ *
+ * @apiSuccess (200: Success) {String[]} Array of ids of users w/ the specified role.
+ * @apiSuccessExample Example Success Response:
+ * 	HTTP/1.1 200 OK
+ *	{
+ *		"data" : ["github44122133", "github22779056", "github5997469", "github98075854"]
+ * 	}
+ *
+ * @apiUse strongVerifyErrors
+ */
+authRouter.get("/roles/list/:ROLE", async (req: Request, res: Response) => {
+	const ROLE: string | undefined = req.params.ROLE?.toString();
```

Comment:
sorry for the confusion! role in your code should be lowercase, but :ROLE should be uppercase.

So something like `const role: string | undefined = req.params.ROLE;